src/components/CodeEditor/CodeEditor.js

```diff
@@ -11,6 +11,12 @@ import {LiveEditor, LiveProvider} from 'react-live';
 import {colors, media} from 'theme';
 import MetaTitle from 'templates/components/MetaTitle';
 
+// Replace unicode to text for other languages
+const unicodeToText = text =>
+  text.replace(/\\u[\dA-F]{4}/gi, match =>
```

Comment:
You can simplify this with:
```
text.replace(/\\u([\dA-F]{4})/gi, (match, p1) => String.fromCharCode(parseInt(p1, 16))
```

Reply:
That's more efficient!. I updated the function.